deb/build/debian/jenkins.service

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=Jenkins Continuous Integration Server
+After=network.target
+
+[Service]
+Type=simple
+User=jenkins
+Group=jenkins
+SyslogIdentifier=jenkins
+UMask='0002'
+LimitNOFILE='8192'
+
+Environment=JENKINS_HOME=/var/lib/jenkins
+ExecStart=/usr/bin/java \
```

Comment:
a lot of other configuration that is in the sysv init is now missing (making things configurable via a file in `/etc/default/jenkins.default`)